dulwich/porcelain.py

```diff
@@ -191,6 +191,87 @@ class RemoteExists(Error):
     """Raised when the remote already exists."""
 
 
+def _parse_timezone_format(tz_str):
```

Comment:
Let's make this public? This should probably be parse_timezone rather than parse_timezone_format, since it's not parsing the format but the timezone itself?

Reply:
I named it ```parse_timezone_format``` because there already is a ```parse_timezone``` in ```objects```, that this very function is using.

Reply:
Made public in 33d9f65866cfffe212bec969f1a5afd79a8ebc7a